tests/test_absorption_spectrum_fit.py

```diff
@@ -0,0 +1,69 @@
+"""
+Unit test for the AbsorptionSpectrum analysis module
+"""
+
+#-----------------------------------------------------------------------------
+# Copyright (c) 2014-2017, yt Development Team.
+# Copyright (c) 2017, Trident Development Team.
```

Comment:
Just a minor thing, but I think you can make the copyright `Copyright (c) Trident Development Team. All rights reserved.` and make it year independent. Also, since it's a new file, we don't need the `yt` part of it.